src/utils/pii-remover.ts

```diff
@@ -60,7 +60,7 @@ const isPhoneNumber = function (value: string) {
 };
 
 export function stripPIIFromString(value: string) {
-  var stripped = value.replace(EMAIL_PATTERN, "[email]");
+  let stripped = value.replace(EMAIL_PATTERN, "[email]");
```

Comment:
because we will are not updating this variable after this line, if we have to change it's better to use CONST instead of let
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
But I think we are, as when I change it to const,
<img width="691" alt="Screenshot 2024-02-22 at 09 04 35" src="https://github.com/govuk-one-login/di-fec-ga4/assets/148252375/1fd8a954-0f95-40d6-96d5-aea79a2916b3">

Reply:
I have changed the others but kept the stripped variable to be initialised with the let keyword

Reply:
oki oki! np!